packages/@aws-cdk/aws-athena/lib/table.ts

```diff
@@ -0,0 +1,102 @@
+import { PolicyStatement } from '@aws-cdk/aws-iam';
+import { IBucket } from '@aws-cdk/aws-s3';
+import { Aws, Construct, IConstruct } from '@aws-cdk/core';
+import { AwsCustomResource } from '@aws-cdk/custom-resources';
+import { DataType } from './datatypes';
+
+/**
+ * Creates a Table with the name and the parameters that you specify.
+ */
+export interface ITable extends IConstruct {
+  /**
+   * The Table name.
+   * @attribute
+   */
+  readonly tableName: string;
+}
+
+/**
+ * Properties used to define an Athena Table
+ */
+export interface TableProps {
+  /**
+   * The database to add the Table to.
+   * @default none
+   */
+  readonly databaseName: string;
+
+  /**
+   * The name for the Table.
+   * @default none
```

Comment:
Do I need '@default' for required properties?

Reply:
No, it doesn't really make sense to have `@default` for required properties - feel free to remove these.